src/api/transaction_test.go

```diff
@@ -1292,7 +1337,8 @@ func TestGetTransactions(t *testing.T) {
 }
 
 type transactionAndInputs struct {
-	txn    coin.Transaction
+	txn coin.Transaction
+	// swagger:ignore
```

Comment:
is it possible to put these comments after field definition ?

Reply:
no, but especially this one, can be eliminated

Reply:
is already deleted for next commit